grpcmock-core/src/main/java/org/grpcmock/InProcessGrpcMockBuilder.java

```diff
@@ -0,0 +1,31 @@
+package org.grpcmock;
+
+import io.grpc.inprocess.InProcessServerBuilder;
+
+import javax.annotation.Nonnull;
+
+/**
+ * @author akoylu
+ */
+public class InProcessGrpcMockBuilder extends GrpcMockBuilder {
+
+  private String name;
+
+  public String getName() {
```

Comment:
Don't think it's a good idea to keep some state and have it only accessible from the builder.